README.md

```diff
@@ -4,8 +4,7 @@
 
 # moodle-tool_objectfs
```

Comment:
Now that we have two storage options may be wise to have a switching object storage section. Should have steps to follow to avoid issues. roughly:

1. Pull all remote objects to duplicated
2. truncate object table and object report table
3. switch

Reply:
Maybe place in a FAQ section that we can add to as needed